pkg/provision/providers/qemu/launch.go

```diff
@@ -461,3 +462,32 @@ func waitForFileToExist(path string, timeout time.Duration) error {
 		time.Sleep(100 * time.Millisecond)
 	}
 }
+
+func checkPartitions(config *LaunchConfig) (diskBootable bool, err error) {
+	f, err := os.Open(config.DiskPaths[0])
```

Comment:
This `f` is not used elsewhere

Reply:
Yep you're right. Below, I was also leaving the closing of the disk to the garbage collector which is not the smartest way to close an open file.